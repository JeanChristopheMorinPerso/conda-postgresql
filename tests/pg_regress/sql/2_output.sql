-- Test the output function
BEGIN;

CREATE EXTENSION conda;

SELECT '1.2.3'::condaversion::text;

ROLLBACK;
