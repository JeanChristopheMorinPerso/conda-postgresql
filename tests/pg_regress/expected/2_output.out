-- Test the output function
BEGIN;
CREATE EXTENSION conda;
SELECT '1.2.3'::condaversion::text;
 text  
-------
 1.2.3
(1 row)

ROLLBACK;
